src/pages/index.tsx

```diff
@@ -1,30 +1,166 @@
+                  <li>
+                    <a href="https://www.gomomento.com/blog/how-to-create-a-slack-workflow-with-webhooks-in-momento-topics" target='_blank'>
+                      <div className={styles.openInNew}>
+                        <span>Publish messages to Slack</span>
+                        <IoOpenOutline />
+                      </div>
+                    </a>
+                  </li>
+                </ul>
+              </div>
+              <div>
+                <h2>For developers</h2>
+                <ul className={styles.columnList}>
+                  <li><a href="/sdks">Available SDKs</a></li>
+                  <li><a href="/cache/develop/authentication">Authentication and access control</a></li>
+                  <li>
+                    <a href="https://aws.amazon.com/marketplace/pp/prodview-ndqotnrhhjyvo" target="_blank">
+                      <div className={styles.openInNew}>
+                        <span>Momento on AWS Marketplace</span>
+                        <IoOpenOutline />
+                      </div>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="https://console.gomomento.com" target="_blank">
+                      <div className={styles.openInNew}>
+                        <span>Try it out in the browser</span>
+                        <IoOpenOutline />
+                      </div>
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <div className={styles.section}>
+            <h2 className={styles.sectionTitle}>Overview</h2>
+            <p>
+              Momento simplifies cloud architecture and automates resource management so developers can build from zero to global scale with no distractions.
+              With our flagship <b>Cache</b> and <b>Topics</b> services, Momento provides developers the building blocks they need to focus on the thing that matters
+              most: <i>your business problem</i>.
+            </p>
+            <div className={styles.columns}>
+              <div className={styles.subsectionDetail}>
+                <h2 className={styles.subsectionTitle}>It's about <i>developer productivity</i></h2>
+                <p>
+                  We want you to move as fast as possible. Whether your database is pushing millions of transactions per second or you're publishing messages to tens of
+                  thousands of subscribers, Momento is with you managing the infrastructure, tackling scaling events, and providing you with simple API calls.
+                </p>
+
+                <h2 className={styles.subsectionTitle}>We go where you go</h2>
+                <p>
+                  Momento works hard to provide developers with SDKs in as many programming languages possible. We have clients for all types of projects: browsers, Android
+                  development, iOS apps, and 10+ server-side languages.
```

Comment:
Should we also put emphasis on the hand-craftiness of the SDKs with respect to Configurations (Lambda/serverless, InRegion, Laptop etc) and the other plumbing stuff?